assets/src/edit-story/elements/media/edit.js

```diff
@@ -107,25 +108,47 @@ function MediaEdit({ element, box }) {
   const [croppedMedia, setCroppedMedia] = useState(null);
   const [cropBox, setCropBox] = useState(null);
   const elementRef = useRef();
+  const [localProperties, setLocalProperties] = useState({});
+  const lastLocalProperties = useRef({});
```

Comment:
Couldn't we use just `localProperties` and skip the `lastLocalProperties`?

Reply:
We need it in a ref to have a stable function to pass to the unmount-function. But we also need it in a state, to make the component re-render on updates. So, we need both here.